pkg/brokerapi/service_instance.go

```diff
@@ -31,16 +31,9 @@ type ServiceInstance struct {
 	Parameters map[string]interface{} `json:"parameters, omitempty"`
 }
 
-// LastOperation represents the state of a discrete action that the broker is
-// completing asynchronously
-type LastOperation struct {
-	State       string `json:"state"`
-	Description string `json:"description,omitempty"`
-}
-
-// ServiceInstanceRequest represents a request to a broker to provision an
+// CreateServiceInstanceRequest represents a request to a broker to provision an
 // instance of a service
-type ServiceInstanceRequest struct {
+type CreateServiceInstanceRequest struct {
```

Comment:
For each verb (create/update/delete), each request object has a slightly different sort of structure. I thought it made sense to make three separate objects to match this, as opposed to jamming all the possible attributes in one mega request object.